client/my-sites/plans/main.jsx

```diff
@@ -139,8 +142,15 @@ const Plans = React.createClass( {
 
 	render() {
 		const selectedSite = this.props.sites.getSelectedSite();
+		let PlanList = OldPlanList;
 		let hasJpphpBundle,
-			currentPlan;
+			currentPlan,
+			mainStyle;
+
+		if ( showPlanFeatures ) {
+			PlanList = PlanFeaturesList;
+			mainStyle = { maxWidth: 1040 };
```

Comment:
I would suggest passing through a class name instead of passing through a style prop. @mtias do you have a suggestion for the modifier class? Looks like default main styling is set to `max-width: 720px;` maybe 'is-wide-layout' ?